### DISCUSSION TITLE
gyroscope spike prime

### DISCUSSION THREAD

**Discussion by Xylar52** [pybricks/support#980] [Q&A] [answered]
i prefer Spike Prime over EV3 for many reasons and the integrated gyroscope is one of them. Is there a way to use the gyroscope for yaw turns ? "imu.tilt" only gives the current angle for pitch and roll. however i can't get or even use yaw axis, which in my opinion i think is the most useful because well that's how things turn when they are moving on the ground

**Comment by dlech** [comment]
We are working on it. Please see https://github.com/pybricks/support/issues/912 and https://github.com/pybricks/support/issues/933.

**Comment by thehomelessguy** [comment]
They are actively working on it. My understanding is that they'd like it to be much more precise than the lego implementation. If you use the new spike app or the legacy lego python app the gyro on the spikes actually is pretty in accurate. I think if I tried to have it do a fairly slow 360 degree turn it'd be easily off by 5-10 degrees. Honestly, once you have it calibrated the DriveBase control is accurate enough I have not missed the lack of gyro functionality.

**Comment by laurensvalk** [comment, marked as answer]
A new answer can be found in https://github.com/pybricks/support/issues/989, so we can close this thread.
